lnwallet/test/test_interface.go

```diff
@@ -1309,13 +1309,31 @@ func testListTransactionDetails(miner *rpctest.Harness,
 
 	// Query for transactions only in the latest block. We do not expect
 	// any transactions to be returned.
-	txDetails, err = alice.ListTransactionDetails(
+	txDetails, _, _, err = alice.ListTransactionDetails(
 		chainTip, chainTip, "", 0, 1000,
 	)
 	require.NoError(t, err, "unexpected error")
 	if len(txDetails) != 0 {
 		t.Fatalf("expected 0 transactions, got: %v", len(txDetails))
 	}
+
+	// Next we try to query for only 5 transactions.
+	var (
+		firstIdx, lastIdx uint64
+	)
+	txDetails, firstIdx, lastIdx, err = alice.ListTransactionDetails(
+		startHeight, chainTip, "", 0, 5,
+	)
+	require.NoError(t, err, "unexpected error")
+	if len(txDetails) != 5 {
```

Comment:
New code should use the `require` library. So this could be `require.Len(t, txDetails, 5)`. And below `require.EqualValues(t, 1, firstIdx)`, and so on.